--- atl-init-confluence.orig	2017-08-16 13:53:42.960729952 -0400
+++ atl-init-confluence	2017-08-16 13:55:31.677002567 -0400
@@ -193,7 +193,9 @@
     atl_log "Creating file /etc/ld.so.conf.d/confluence.conf"
     echo /usr/lib/jvm/jre-1.7.0-openjdk.x86_64/lib/amd64/server/ > /etc/ld.so.conf.d/confluence.conf
     sudo ldconfig
-    service collectd restart
+    if [[ "xtrue" == "x$(atl_toLowerCase ${ATL_COLLECTD_ENABLED})" ]]; then
+      service collectd restart
+    fi
     atl_log "Creating file /etc/ld.so.conf.d/confluence.conf ==> done"
 
     if [[ -d "${ATL_CONFLUENCE_INSTALL_DIR}" ]]; then
