--- atl-init-synchrony.orig	2017-08-17 11:52:31.799723633 -0400
+++ atl-init-synchrony	2017-08-17 11:51:38.157555010 -0400
@@ -185,7 +185,9 @@
     atl_log "Creating file /etc/ld.so.conf.d/confluence.conf"
     echo /usr/lib/jvm/jre-1.7.0-openjdk.x86_64/lib/amd64/server/ > /etc/ld.so.conf.d/confluence.conf
     sudo ldconfig
-    service collectd restart
+    if [[ "xtrue" == "x$(atl_toLowerCase ${ATL_COLLECTD_ENABLED})" ]]; then
+        service collectd restart
+    fi
     atl_log "Creating file /etc/ld.so.conf.d/confluence.conf ==> done"
 
     if [[ -d "${ATL_CONFLUENCE_INSTALL_DIR}" ]]; then
